view/js/main.js

```diff
@@ -33,6 +33,50 @@ function _resizeIframe(obj, desth) {
 	setTimeout(_resizeIframe, 100, obj, ch);
 }
 
+function getCookie(name) {
```

Comment:
Please use `localStorage` instead of cookies. See https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API

Reply:
Done